### PR TITLE
OPDS fixes

### PR DESCRIPTION
**Pull request by hius07**
(1) No more special calibre entry, local catalogs to be added in usual way. Instead of https://github.com/koreader/koreader/pull/10623.
(2) Fix OPDSPSE hack of ImageViewer.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/10657)

### CODE REVIEW

File: frontend/ui/data/onetime_migration.lua [hunk 1]
```diff
@@ -566,5 +567,24 @@ if last_migration_date < 20230703 then
     end
 end
 
+-- 20230707, OPDS, no more special calibre catalog
+if last_migration_date < 20230707 then
+    logger.info("Performing one-time migration for 20230707")
+
+    local calibre_opds = G_reader_settings:readSetting("calibre_opds")
+    if next(calibre_opds) then
```
Comment (poire-z) [inline frontend/ui/data/onetime_migration.lua:575]: If that plugin happened to have been disabled (at install time, for some reason, or what do we know why :), you may get calibre_opds = nil, so next() will crash. Better check for nil.

**Comment by hius07** [comment]
Mark catalogs with a user logged-in.

**Comment by poire-z** [comment]
Was going to write the icons image shouldn't be a lock image, as the user is unlocked as it has entered a username - but at a second look, it looks like these icons are not a lock but a person
image :)

**Comment by hius07** [comment]
Yes, it's a "user": https://fontawesome.com/v4/icon/user-o

**Comment by NiLuJe** [comment]
I like it ;).

**Review by NiLuJe — APPROVED** [review]
Looks good at a quick glance ;).

**Pull request merged**